src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/RestrictedPatientsService.kt

```diff
@@ -73,6 +75,49 @@ class RestrictedPatientsService(
       .let { addRestrictedPatient(it) }
   }
 
+  @Transactional
+  fun prisonerReleased(offenderNo: String) {
+    if (restrictedPatientsRepository.existsById(offenderNo)) {
+      log.info("Ignoring release of prisoner $offenderNo as they are already a restricted patient")
```

Comment:
I was getting warnings at one point in intellij about this sort of code - telling me that `offenderNo` should be passed as a parameter instead